src/data/parsing.py

```diff
@@ -0,0 +1,309 @@
+import argparse
+import gc
+import json
+import pickle
+
+import spacy
+import stanfordnlp
+from interruptingcow import timeout
+
+import equationparsing as eq
+import textparsing

```

Comment:
This is better off being `from textparsing import clean_doc` as it's more explicit as to what you're bringing in.